README.rst

```diff
@@ -147,6 +147,29 @@ If you have external files you can add them in ``examples`` folder to skip.
 
 For rst-syntax-error skip unknown directives
 
+Skip one check based on the Odoo version
+--------------
+
+If you need to valid one check using the Odoo version you should be used the
```

Comment:
If you need to restrict a check to specific version(s) of Odoo, use the new variable ...